### PR TITLE
test: start FP in op e2e

### PR DESCRIPTION
**Pull request by bap2pecs**
## Summary

start FP in e2e test

also found two problems"

1. incorrect wasm binary was used. it might be overwritten when I was fixing merge conflicts. a better way is to append the binary with the commit hash where I built from in https://github.com/babylonchain/babylon-contract
2. `go list` can't find any test b/c tag was not added

## Test Plan

```
make test-e2e-op
```

wait for CI

### CODE REVIEW

**Review by lesterli — APPROVED** [review]

**Comment by bap2pecs** [comment]
@gusin13 I noticed that there is a version.txt file you added in https://github.com/babylonchain/finality-provider/pull/374

any tip how we should update it given that we also have our own bytecode added?

File: Makefile [hunk 1]
```diff
@@ -83,11 +84,11 @@ clean-e2e:
 test-e2e: install-babylond install-wasmd
 	make clean-e2e
 	go test -mod=readonly -timeout=25m -v $(PACKAGES_E2E) -count=1 --tags=e2e
-	go test -mod=readonly -timeout=25m -v $(PACKAGES_E2E) -count=1 --tags=e2e_op
+	go test -mod=readonly -timeout=25m -v $(PACKAGES_E2E_OP) -count=1 --tags=e2e_op
```
Comment (maurolacy) [inline Makefile:87]: Is this correct? Looks like a bug to me.

Comment (bap2pecs) [reply Makefile:87]: which part looks wrong? I added some explanation in https://github.com/babylonchain/finality-provider/issues/415

Comment (maurolacy) [reply Makefile:87]: Shouldn't the env var be `PACKAGES_E2E` here? I mean, these tests are the e2e tests, not the e2e-op ones.

Perhaps I'm missing something.

Comment (bap2pecs) [reply Makefile:87]: `test-e2e` is supposed to test all tests including e2e-op ones

Comment (maurolacy) [reply Makefile:87]: Got it, there's another `go test` above.

**Pull request merged**